android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.kt

```diff
@@ -82,7 +84,9 @@ class ScreenStackHeaderConfig(context: Context) : ViewGroup(context) {
         // we want to save the top inset before the status bar can be hidden, which would resolve in
         // inset being 0
         if (headerTopInset == null) {
-            headerTopInset = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
+            headerTopInset = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R)
```

Comment:
Do you know if `VERSION_CODES.R` is already defined on lower supported Android versions?

Reply:
I don't think it is, but it should be converted to a number during build time so it does not matter according to this thread: https://stackoverflow.com/questions/21874227/why-do-android-os-build-version-codes-work-on-older-platforms

Reply:
Yeah, but where does the guarantee that this symbol is available in build time comes from?

Reply:
We have this guarantee only since August 2023.

I'm leaving link here for future git-blaming activities.

Reply:
Actually we don't have this guarantee, cause this link is about `target`. I guess we can reasonably safely assume that `target` & `compile` point to the same version 🤷🏻